### DISCUSSION TITLE
Can't get code to clone repository.

### DISCUSSION THREAD

**Discussion by roaddog4** [Comfy-Org/ComfyUI#4062] [Q&A]
When I click the Green Code button, it turns bright green and has a white outline around it and nothing else appears. Do I have a setting wrong in windows or something else?  How do i get the git link to install the repository?

**Comment by comfyanonymous** [comment]
You can just use the url of the repo: https://github.com/comfyanonymous/ComfyUI.git


However if you are not familiar with git I recommend you grab the download from the "Direct link to download" on: https://github.com/comfyanonymous/ComfyUI#installing
